### PR TITLE
Revise Code of Conduct

### PR DESCRIPTION
**Pull request by russss**
This adds a bit more detail on how we will respond to CoC issues, as
well as some other miscellaneous tweaks.

### CODE REVIEW

File: templates/code-of-conduct.html [hunk 1]
```diff
@@ -3,41 +3,60 @@
...
-  All our volunteer staff will be briefed on how to handle a harassment situation.</li>
+  All our volunteer staff will be briefed on how to handle reports.</li>
   <li>Drop us an email at <a href="mailto:{{ config['DUTY_EMAIL'][1] }}">{{ config['DUTY_EMAIL'][1] }}</a>.</li>
 </ul>
-<p>If you have any comments or queries, you can get in touch with our Conduct &amp; Accessibility team at <a href="mailto:{{ config['CONDUCT_EMAIL'][1] }}">{{ config['CONDUCT_EMAIL'][1] }}</a>.</p>
 
-<p><small>This Code of Conduct is licensed under the <a href="https://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 License</a>.</small></p>
+<h3>Our Response</h3>
+
+<p>During our events, we aim to deal with all conduct issues within 12 hours.
```
Comment (Jonty) [inline templates/code-of-conduct.html]: Is it worth specifying the response time when we're not running an event? (Apols for slow reply)

Comment (russss) [reply templates/code-of-conduct.html]: Ideally yes, but I don't want to make promises we can't keep, so I have no idea what timeframe would be reasonable.

Comment (Jonty) [reply templates/code-of-conduct.html]: Is "deal with" implying that we'll finish handling with the issue within that 12 hour period, or just that our initial response will be within 12 hours?

Comment (russss) [reply templates/code-of-conduct.html]: Good point, that's not terribly clear. I changed it to "reach a decision".

Comment (Jonty) [reply templates/code-of-conduct.html]: In which case we should say something like "outside of events, we aim to initially respond within one week".

I don't feel comfortable committing to a resolution time outside events, but I feel we can at least say we'll respond saying "we have received your email" within a week or so.

Comment (russss) [reply templates/code-of-conduct.html]: I think that's fair, I'll add that. I'd like to be able to promise a shorter time than that but we need to make sure we'll meet it.

**Pull request merged**